llama_stack/models/llama/sku_list.py

```diff
@@ -22,6 +22,20 @@ def resolve_model(descriptor: str) -> Model | None:
     for m in all_registered_models():
         if descriptor in (m.descriptor(), m.huggingface_repo):
             return m
+
+    # Check provider aliases by attempting to import and check common providers
+    try:
+        from llama_stack.providers.remote.inference.together.models import MODEL_ENTRIES as TOGETHER_ENTRIES
```

Comment:
not sure why we need to do this. why do we need to pick out any provider(s) as a common provider?